### DISCUSSION TITLE
Why is Webots running so slow on my computer?

### DISCUSSION THREAD

**Discussion by omichel** [cyberbotics/webots#5401] [Q&A]
The provided example simulations are running very slowly. How can I improve this?

**Comment by omichel** [comment]
You should verify your graphics driver installation. Please find instructions in [this section](https://www.cyberbotics.com/doc/guide/verifying-your-graphics-driver-installation).

If you are using a laptop computer, please check the power options and make sure you are using the high performance power plan.

On Ubuntu (or other Linux) we also recommend to disable all visual effects. Depending on the graphics hardware, there may be a huge performance drop of the rendering system (up to 10x) when the visual effects are on. You can easily disable the operating system's visual effects using some tools like Compiz Config Settings Manager or Unity Tweak Tool.

**Comment by zydczqcnydx** [comment]
How to set it up specifically Compiz

**Reply by ShuffleWire** [reply]
Did you end up using it or do you need help ?
